python/src/DistributionImplementation_doc.i.in

```diff
@@ -15,13 +15,13 @@ OT_Distribution_doc
 
 Parameters
 ----------
-X : sequence of float, 2-d sequence of float
-    CDF input(s).
+x : sequence of float, 2-d sequence of float
```

Comment:
These updates are interesting, but they are not associated to the non linear equation solver that this PR is about. I suggest to create a separate PR to discuss them.